psalm.xml

```diff
@@ -1,15 +1,11 @@
 <?xml version="1.0"?>
 <psalm
-    errorLevel="6"
-    resolveFromConfigFile="true"
+    errorLevel="2"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xmlns="https://getpsalm.org/schema/config"
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
 >
     <projectFiles>
         <directory name="src" />
-        <ignoreFiles>
```

Comment:
Ignore vendor is userful. In some cases psalm follow from package code to vendor and check him.

Reply:
Alright